rero_ils/modules/acq_accounts/serializers/json.py

```diff
@@ -17,12 +17,9 @@
 
 """Acquisition account serialization."""
 
-from invenio_records_rest.serializers.response import record_responsify, \
-    search_responsify
-
-from .api import AcqAccountsSearch
-from ..libraries.api import Library
-from ..serializers import JSONSerializer, RecordSchemaJSONV1
+from rero_ils.modules.acq_accounts.api import AcqAccountsSearch
```

Comment:
COSMETIC : This import could stay "relative" because we use a class from the same resource.